tests/test_expt_file_counts_handler.py

```diff
@@ -0,0 +1,67 @@
+"""
+Copyright 2023 NOAA
+All rights reserved.
+
+Unit tests for expt_file_counts.py
+
+"""
+import os
+import pathlib
+import pytest
+import json
+
+
+from expt_file_counts import ExptFileCountRequest
+
+def test_put_expt_file_counts_request_dict():
+    request_dict = {
+        'name': 'expt_file_counts',
+        'method': 'PUT',
+        'body': {
+            'experiment_name': 'C96L64.UFSRNR.GSI_3DVAR.012016',
+            'wallclock_start': '2021-07-22 09:22:05',
+            'file_type_name': 'example_type',
+            'file_extension': '.example',
+            'time_valid' : '2023-02-03 04:54:23',
+            'storage_loc_name' : 's3_example_bucket',
+            'storage_loc_platform': 'aws_s3',
+            'storage_loc_key': 'noaa-exmample-score-db-bucket/reanalysis',
```

Comment:
it stand out to me that `storage_loc_key` and `folder_path` have a lot of overlap. Is that a bug or a feature?

Reply:
for me it's a feature in that the key is the top level and the folder path was the full path to the specific file being counted. the key is a value that is stored in the storage locations table and used to identify which storage location the file count is associated with whereas the folder path is only in relation to this specific count object.

Reply:
ok. boto3 (aws python lib) usually requires bucket and key to be separate. E.g. technically, for aws, `noaa-exmample-score-db-bucket` is not part of the key.